packages/salesforcedx-utils-vscode/test/jest/context/workspaceContextUtil.test.ts

```diff
@@ -17,32 +24,30 @@ describe('WorkspaceContext', () => {
   let getUsernameStub: jest.SpyInstance;
   let getUsernameOrAliasStub: jest.SpyInstance;
   let workspaceContextUtil: any; // TODO find a better way
-  let authUtil: any;
 
-  const mockWatcher = {
-    onDidChange: jest.fn(),
-    onDidCreate: jest.fn(),
-    onDidDelete: jest.fn()
-  };
+  let mockWatcher: { onDidChange: any; onDidCreate: any; onDidDelete: any; };
   let mockFileSystemWatcher: jest.SpyInstance;
+  let reloadConfigAggregatorsMock: jest.SpyInstance;
 
   beforeEach(async () => {
+    mockWatcher = {
+      onDidChange: jest.fn(),
+      onDidCreate: jest.fn(),
+      onDidDelete: jest.fn()
+    };
+    reloadConfigAggregatorsMock = jest.spyOn(ConfigAggregatorProvider.prototype, 'reloadConfigAggregators');
```

Comment:
note we're testing workspaceUtils here so we want to make out the `reloadConfigAggregators` since it's not the concern of the file under test.